buildrelease.sh

```diff
@@ -48,13 +48,31 @@ done
 cd docs
 ./builddocs.sh root $projects
 
-echo "Release build complete for the following projects:"
+cd ..
+rm -rf releasedocs
+git clone https://github.com/GoogleCloudPlatform/google-cloud-dotnet.git releasedocs -c core.autocrlf=input
```

Comment:
This would be better using `-b gh-pages --depth 1` (and then remove the git checkout below). We don't need any other branches, or full history.

Reply:
Done